src/components/ebay-combobox/README.md

```diff
@@ -0,0 +1,74 @@
+# ebay-combobox
+
+_**For Desktop only**_
```

Comment:
Very minor, but I think it's no longer desktop only.

Reply:
Let me test this quickly before I remove this.

Reply:
Appears to work well on mobile (iOS), but long option lists might pose an issue there. We could address this through JS at a later time if it becomes an issue, but presently is low priority in my book.

Reply:
Hence, removing this statement.

Reply:
Agreed!